airflow/ui/src/pages/DagsList/DagsList.tsx

```diff
@@ -120,6 +120,7 @@ const cardDef: CardDef<DAGResponse> = {
 export const DagsList = () => {
   const [searchParams, setSearchParams] = useSearchParams();
   const [display, setDisplay] = useState<"card" | "table">("card");
+  const [selectedTags, setSelectedTags] = useState<Array<string>>([]);
```

Comment:
Let's store this inside of the search params so that the state persists through page refreshes.

Reply:
Thanks for pointing it out, I updated the data flow to make it get/set from `searchParams`.